VargasMartinezJose/Dis45/HackerRank/ejerciciojava.java/JavaStdinStdout1.java

```diff
@@ -0,0 +1,22 @@
+import java.util.*;
+import java.util.Scanner;
+
+public class JavaStdinStdout1{
+
+    public static void main(String[] args) {
+        
+        Scanner scan = new Scanner(System.in);
+       
+
+        int a = scan.nextInt();
```

Comment:
Please rename your variables for something more meaningful
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html